revocation.ipldsch

```diff
@@ -0,0 +1,10 @@
+type RevocationAction <: Action {
+  cmd "ucan/revoke"
+  nnc ""
+  arg RevocationArguments
+}
+
+type RevocationArguments struct {
+  rev &Delegation
+  pth [&Delegation]
+}
```

Comment:
For the reference our currently deployed implementation uses `ucan` and `proof` field names as opposed to `rev` and `pth` see 

https://github.com/web3-storage/w3up/blob/9aba8a155cca663387952fc9b11bc3748de8f2fc/packages/capabilities/src/ucan.js#L43-L62

Unless there is good reason to introduce a break here I would prefer not to, that said it's not a deal breaker just inconvenient.